scripts/update-release-notes/changelogsAndTags.ts

```diff
@@ -49,28 +51,44 @@ export function getTagToChangelogMap(maxAgeDays?: number): Map<string, IChangelo
  * @returns List of tags
  */
 export function getTags(maxAgeDays?: number): string[] {
+  const ONE_MEGABYTE = 1024 * 1000;
+  const TEN_MEGABYTES = ONE_MEGABYTE * 10;
   console.log(`Getting tags${maxAgeDays ? ` up to ${maxAgeDays} days old` : ''}...`);
 
-  const cmd = 'git for-each-ref --sort=-creatordate --format="%(refname:short) -- %(creatordate)" refs/tags';
+  try {
+    const cmd = 'git for-each-ref --sort=-creatordate --format="%(refname:short) -- %(creatordate)" refs/tags';
+    const gitForEachRefBuffer = execSync(cmd, {
+      // execSync buffer is by default 1MB (node 16). Our git refs tog is much bigger than that, thus setting for 10MB
+      maxBuffer: TEN_MEGABYTES,
+    });
+    const currentBufferSize = (gitForEachRefBuffer.byteLength / ONE_MEGABYTE).toFixed(2);
+
+    console.warn(`
+  📣 NOTE: "git for-each-ref" current buffer size is ${currentBufferSize}MB.
+  If this will be more than maxBuffer ${TEN_MEGABYTES}MB this command will fail and you'll have to increase maxBuffer!
+  `);
 
-  let tagsAndDates = execSync(cmd, { cwd: process.cwd() })
-    .toString()
-    .split(/\r?\n/g)
-    .map(tag => tag.split(' -- '))
-    .filter(arr => arr.length === 2);
+    let tagsAndDates = gitForEachRefBuffer
+      .toString()
+      .split(/\r?\n/g)
+      .map(tag => tag.split(' -- '))
+      .filter(arr => arr.length === 2);
 
-  if (maxAgeDays) {
-    const endIndex = tagsAndDates.findIndex(([, date]) => !_isNewEnough(date, maxAgeDays));
-    if (endIndex !== -1) {
-      tagsAndDates = tagsAndDates.slice(0, endIndex);
+    if (maxAgeDays) {
+      const endIndex = tagsAndDates.findIndex(([, date]) => !_isNewEnough(date, maxAgeDays));
+      if (endIndex !== -1) {
+        tagsAndDates = tagsAndDates.slice(0, endIndex);
+      }
     }
-  }
 
-  const tags = tagsAndDates.map(([tag]) => tag);
+    const tags = tagsAndDates.map(([tag]) => tag);
 
-  console.log(`Found ${tags.length} tag(s).\n`);
+    console.log(`Found ${tags.length} tag(s).\n`);
 
-  return tags;
+    return tags;
+  } catch (err) {
```

Comment:
other errors could happen here right? is it possible to filter this error a bit instead of assuming max buffer reached

Reply:
i wanted to do that but forgot to push lol. ty!